cnd/src/db/models.rs

```diff
@@ -60,3 +70,101 @@ where
         Ok(SwapId(uuid))
     }
 }
+
+#[repr(i32)]
```

Comment:
`repr` is only used for alignment, I think we should not abuse this here. I know that the diesel docs do it as well but I think it is wrong.

Better add an explicit method that returns an integer for your variant if you need that:

```rust
impl Role {
   fn index(&self) -> u32 {
      match self {
          Role::Alice => 0,
          Role::Bob => 1,
	  }
   }
}
```

Reply:
However, as mentioned, I think they should be strings.

Reply:
Cool, thanks for the review.

Reply:
Implemented as suggested.  Discussed use of macro to generate the `to_sql` and `from_sql` trait implementations for enums.  Was only able to do `to_sql` - `from_sql` should be possible with some extra thought.  Leaving for another time.

Reply:
If a way to do the macro jumps out at anyone feel free to say and I'll implement it.